From f731e1d7354d5fc0fe86f9386fa7c547543f8dbf Mon Sep 17 00:00:00 2001
From: David S. Wang <dsw@cloudera.com>
Date: Wed, 27 Mar 2013 08:49:10 -0700
Subject: [PATCH 06/96] HBASE-6606 Test for reconnecting with HBaseAdmin using unmanaged HConnection

Reason: Supportability
Author: Gregory Chanan
Ref: CDH-5549
(cherry picked from commit 2ff1e098e69fc3a7500a717c1c3c2ffd14c31402)
---
 .../hadoop/hbase/client/TestFromClientSide.java    |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide.java b/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide.java
index aa0a7f2..4a32441 100644
--- a/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide.java
+++ b/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide.java
@@ -3907,6 +3907,7 @@ public class TestFromClientSide {
     return new HTable(tableName, conn, pool);
   }
 
+  
   /**
    * simple test that just executes parts of the client
    * API that accept a pre-created HConnction instance
-- 
1.7.0.4

